editors/code/src/main.ts

```diff
@@ -115,76 +116,86 @@ async function bootstrapExtension(config: Config, state: PersistentState): Promi
 	await vscode.commands.executeCommand("workbench.action.reloadWindow");
 }
 
-async function bootstrapServer(config: Config, state: PersistentState): Promise<string> {
-	const path = await getServer(config, state);
+async function bootstrapServer(): Promise<string> {
+	const path = await getServer();
 	if (!path) {
 		throw new Error(
-			"Clarity Language Server is not available. " +
-			"Please, ensure its [proper installation](https://github.com/hirosystems/clarity-lsp)."
+			"Clarinet is not available.\n" +
+			"Please ensure it is correctly [installed](https://github.com/hirosystems/clarinet)"
 		);
 	}
 
-	// log.debug("Using server binary at", path);
-
-	// const res = spawnSync(path, ["--version"], { encoding: 'utf8' });
-	// log.debug("Checked binary availability via --version", res);
-	// log.debug(res, "--version output:", res.output);
-	// if (res.status !== 0) {
-	//     throw new Error(`Failed to execute ${path} --version`);
-	// }
+	const res = spawnSync(path, ["--version"], { encoding: 'utf8' });
+	log.error("Checked binary availability via --version", res);
+	log.debug(res, "--version output:", res.output);
+	// Yikes: `$ clarinet --version` returns
+	// clarinet 0.21.0
+	//
+	// The LSP was merged in Clarinet in v0.21.0, we want to make sure that 
+	// we're using an adequate version.
+	const version = res.output
+		.toString()    // clarinet 0.21.0
+		.split("\n")[0] // 0.21.0
+		.split(" ")[1] // 0.21.0
+		.split(".");   // ["0", "21", "0"]
+	if (parseInt(version[0]) === 0 && parseInt(version[1]) < 22) {
+		throw new Error(
+			"Clarinet is outdated.\n" +
+			"Please update to [v0.22.0 or newer](https://github.com/hirosystems/clarinet)"
```

Comment:
Should this say "v0.21.0 or newer"?

Reply:
`v0.21.0` would work, but this version is not that great because of the bug fixed here https://github.com/hirosystems/clarinet/pull/196